custom/logistics/api.py

```diff
@@ -234,15 +234,15 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
                 user.is_active = bool(ilsgateway_smsuser.is_active)
                 user.user_data = user_dict["user_data"]
                 if "phone_numbers" in user_dict:
-                    user.set_default_phone_number(user_dict["phone_numbers"][0])
+                    user.set_default_phone_number(apply_leniency(user_dict["phone_numbers"][0]))
                     try:
-                        user.save_verified_number(self.domain, user_dict["phone_numbers"][0], True,
-                                                  ilsgateway_smsuser.backend)
+                        user.save_verified_number(self.domain, user_dict["phone_numbers"][0], True)
                     except PhoneNumberInUseException as e:
                         v = VerifiedNumber.by_phone(user_dict["phone_numbers"][0], include_pending=True)
                         v.delete()
-                        user.save_verified_number(self.domain, user_dict["phone_numbers"][0], True,
-                                                  ilsgateway_smsuser.backend)
+                        user.save_verified_number(self.domain, user_dict["phone_numbers"][0], True)
+                    except InvalidFormatException:
+                        pass
```

Comment:
I don't know where this method is being called from, but is it possible to show this error to the user who initiates this workflow rather than passing on it?  Or is the phone number validated beforehand so this should never happen?